package.json

```diff
@@ -14,20 +14,37 @@
   },
   "license": "MIT",
   "dependencies": {
-    "hoek": "2.x.x"
-  },
-  "peerDependencies": {
-    "hapi": ">= 8.x.x < 10"
+    "autoprefixer-stylus": "0.8.0",
```

Comment:
n.x.x for hapi/CL, n.n.x when version >= 1. n.n.n for < 1. We've covered this in the past. If you've published to npm, and you're < 1, you don't get semver so we can't trust you at 0.n.n